ksp_plugin/pile_up.cpp

```diff
@@ -424,27 +426,40 @@ DegreesOfFreedom<Barycentric> PileUp::RecomputeFromParts(
   // Then compute the inertia tensor and the angular momentum of the pile-up.
   InertiaTensor<RigidPileUp> pile_up_inertia_tensor;
   Bivector<AngularMomentum, RigidPileUp> pile_up_angular_momentum;
```

Comment:
Using this type to compute the angular momentum only makes sense if `RigidPileUp` remains non-rotating, as it is in non-rotating frames that the quantity ∫ r ∧ v dm is conserved (and is called the angular momentum).

Reply:
But `RigidPileUp` *is* non-rotating.  I have fixed the comment in `frames.hpp`.